freespace2/freespace.cpp

```diff
@@ -1272,6 +1272,8 @@ void freespace_mission_load_stuff()
 	}
 }
 
+extern void mission_process_alt_types();
```

Comment:
Please define this in the relevant header.

Reply:
Done.